ansible/scripts/init.sh

```diff
@@ -18,3 +18,11 @@ CUR_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
 INVENTORY_DIR="${CUR_DIR}/../inventory"
 PLAYBOOKS_DIR="${CUR_DIR}/../playbooks"
+
+check_inventory() {
+	inv_dir=$(dirname $1)
+	if [ ! -e "${inv_dir}/group_vars" ]; then
+		echo "Inventory directory ${inv_dir} is missing group_vars directory"
+		exit 1
+	fi
+}
```

Comment:
No. However, the ``inventory`` is known after you set it. And that is after you run the ``./init.sh``. I can create another function, e.g. ``ansible_playbook``, that will call ``check_inventory`` and then ``ansible-playbook``.